src/controllers/public/images-controller.js

```diff
@@ -12,15 +12,18 @@ module.exports = {
 
     return res.json(result);
   },
-  async updateImage(req, res) {
-    const result = await imagesService.update(req.params.id, req.body);
+  async getPublicImagesPagination(req, res, next) {
```

Comment:
Why this func named "getPublicImages**Pagination**" ? This func still returns a list of images as main part of result.